### PR TITLE
Fix pylint

### PR DESCRIPTION
**Pull request by nikita-klsh**
This PR fixes pylint output in the latest docker image analysiscenter1/ds-py3. 

A lot of `missing-function-docstring` and `abstract-method` warnings happened in `batchflow/models/torch` folder we cant fix, because they occur on the torch side. They are fixing [this](https://github.com/pytorch/pytorch/issues/42305) and [this](https://github.com/pytorch/pytorch/issues/43057). 

So we can:
1.  Close eyes on this warnings.
2.  Temporary set additional restrictions for pylint for `torch` folder only like its currently done. 

@roman-kh

### CODE REVIEW

**Comment by roman-kh** [comment]
@nikita-klsh It seems `pytorch` issues are solved already.

**Comment by nikita-klsh** [comment]
> @nikita-klsh It seems `pytorch` issues are solved already.

@roman-kh There are no releases with updates yet. Or we want to build it from github?

**Comment by nikita-klsh** [comment]
Running `pylint` with `torch 1.7.0` still raises warnings about `forward` methods missing docs. 
Torch solved the issue and now the method formally obtains the doc, i.e. `nn.Module.forward.__doc__ is not None`, even tho docs are not explicitly speсified in the method's body. 
I suppose this might be `pylint` false warning.

File: batchflow/dataset.py [hunk 1]
```diff
@@ -75,7 +75,6 @@ def __init__(self, index, batch_class=Batch, *args, preloaded=None, cast_to_arra
         self._attrs = None
         kwargs['_copy'] = kwargs.get('_copy', copy)
         self.n_splits = None
-        self.train, self.test = None, None
```
Comment (roman-kh) [inline batchflow/dataset.py]: Why do you think this is a better approach?

Comment (nikita-klsh) [reply batchflow/dataset.py]: Looks weird to reinitialize attributes, that have already been initialized in the parent class `Baseset`

Comment (nikita-klsh) [reply batchflow/dataset.py]: I return it as it was initially and plug `pylint: disable=attribute-defined-outside-init` stub.
In the code we assign values to `self.train / self.test / self.validation` attributes that WERE defined in the `init`.
It looks like pylint false positive

**Pull request merged**